salsa_dancing_molecules/variables.py

```diff
@@ -33,15 +35,26 @@ def list_to_array(self):
         self.atomic_pressures = np.array(self.atomic_pressures, dtype=float)
         self.temperatures = np.array(self.temperatures, dtype=float)
         self.specific_heat_NVE = np.array(self.specific_heat_NVE, dtype=float)
+        self.timesteps = np.array(self.timesteps, dtype=float)
 
     def generate_file(self, file_name='simulation_data.csv'):
         """Save the physical quantatives into a CSV file."""
         file_saver(
             output_path=file_name,
             data_head=['Potential Energy (eV)', 'Kinetic Energy (eV)',
                        'Pressure (Pa)', 'Temperature (K)',
-                       'Specific Heat Capacity NVE (J/[K kg])'],
+                       'Specific Heat Capacity NVE (J/[K kg])',
+                       'Time (fs)'],
             data_list=[self.potential_energies, self.kinetic_energies,
                        self.atomic_pressures, self.temperatures,
-                       self.specific_heat_NVE]
+                       self.specific_heat_NVE, self.timesteps]
         )
+
+    def set_timestep(self):

```

Comment:
Let set_timestep() take and save the time step size, then add a function called "increment_time()" that performs the increment.